### ISSUE TITLE
Warn about the distance in the documentation

### ISSUE THREAD

**Issue by louismaximepiton** [Orange-OpenSource/Orange-Boosted-Bootstrap#3810]
### Prerequisites

- [x] I have [searched](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed feature requests
- [x] I have read the [contributing guidelines](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/.github/CONTRIBUTING.md)

### Proposal

Add a callout in the doc, probably in the accessibility and Max-width paragraph to tell people the max-width shouldn't be too wide and explicitely tell that the max-width should be used when not used inside a container.
It should be there for every component that has a max-width.

### Motivation and context

Better take into account the a11y inside our doc
